ci/test-zk.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# Colors
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m'
+
+# Flags and default modes
+INSTALL_MODE="preinstalled"
+ALL_FILES=false
+
+# Parse flags
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        --all) ALL_FILES=true; shift ;;
+        --install-zkwasm-permanent) INSTALL_MODE="permanent"; shift ;;
+        --install-zkwasm) INSTALL_MODE="temporary"; shift ;;
+        --help)
+            echo "Usage: $0 [OPTIONS] [filename.zkasm]"
+            echo "Options:"
+            echo "  --all                           Test all zkasm files"
+            echo "  --install-zkwasm                Temporarily install and use zkevm-rom"
+            echo "  --install-zkwasm-permanent      Permanently install zkevm-rom"
+            echo "  --help                          Show this message"
+            exit 0
+            ;;
+        *) break ;;
+    esac
+done
+
+if [ "$ALL_FILES" = false ] && [ -z "$1" ]; then
+    echo "Please provide a filename or use the --all flag to test all files."
+    exit 1
+fi
+
+
+BASE_DIR="../wasmtime"
+
+case $INSTALL_MODE in
+    "permanent")
+        echo "Cloning zkevm-rom into ../../ directory..."
+        git clone https://github.com/0xPolygonHermez/zkevm-rom/ ../../zkevm-rom > /dev/null 2>&1
+        cd ../../zkevm-rom
+        npm install
+        ;;
+    "temporary")
```

Comment:
I don't think we would ever use this mode, both in local and remote context, should we remove it?